ci-operator/config/stackrox/stackrox/stackrox-stackrox-release-3.74.yaml

```diff
@@ -23,7 +23,7 @@ build_root:
   image_stream_tag:
     name: apollo-ci
     namespace: stackrox
-    tag: stackrox-ui-test-0.4.3
+    tag: stackrox-ui-test-0.4.4
```

Comment:
did we even need to update the Go version for 3.74 in the first place? I believe we do not need to fix any vulnerabilities for 3.74 at this point, so I wonder if it's best to keep this on whatever version of Go it was originally on?